lib/client.js

```diff
@@ -54,8 +98,12 @@ Freshdesk.prototype = Object.create({
 		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/tickets`, params, null, cb)
 	},
 
-	listAllTicketFields(cb) {
-		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/ticket_fields`, null, null, cb)
+	listAllTicketFields(options, cb) {
+		const qs = options.type
```

Comment:
existing 3d-party code will fail after this update.
You shoud perform type check, when there is just one argument. Or something else.

*example of code which will fail:

```js
const callback  = function(x,y,z) { console.log(x,y,z); }
listAllTicketFields(callback)
```

After update `callback` won't be called.